src/interpolations.jl

```diff
@@ -489,6 +503,25 @@ function reference_coordinates(::Lagrange{RefLine,1})
             Vec{1, Float64}(( 1.0,))]
 end
 
+"""
+Mapping from 1D line to point.
+"""
+function transfer_point_cell_to_face(point::Union{Vec{1, T}, Vector{T}}, cell::Type{RefLine}, face::Int) where T
```

Comment:
1) Perhaps use regular comments, `#` above each specialization to avoid getting all these as docstrings. The general descriptions on the top should suffice IMO.
2) Is it necessary to support `Vector` as input? Do we use that?

Reply:
The `Vector` was used in #743 IIRC, I'll check if it can be removed (and probably it can, I think I changed its calls in #743 but didn't remove `Vector`)